### PR TITLE
Convert two functions to const fn

### PR DESCRIPTION
**Pull request by rzumer**
Work towards #1464.

### CODE REVIEW

File: src/tiling/tiler.rs [hunk 1]
```diff
@@ -118,8 +118,7 @@ impl TilingInfo {
     k
   }
 
-  #[inline(always)]
-  pub fn tile_count(&self) -> usize {
+  pub const fn tile_count(&self) -> usize {
```
Comment (lu-zero) [inline src/tiling/tiler.rs]: cols and rows aren't constant, the compiler shouldn't allow this.

Comment (rzumer) [reply src/tiling/tiler.rs]: It does though.

Comment (tdaede) [reply src/tiling/tiler.rs]: I think it just means you can construct a const TilingInfo, and then call this method on it and the result is also const.

Comment (rzumer) [reply src/tiling/tiler.rs]: That makes sense.

Comment (rzumer) [reply src/tiling/tiler.rs]: @lu-zero I added back the inline attribute to this function, let me know if that's fine by you.

Comment (lu-zero) [reply src/tiling/tiler.rs]: Sure, thank you :)

**Pull request merged**